types/core/config.d.ts

```diff
@@ -8,16 +8,16 @@ declare const CONFIG: {
    */
   Actor: {
     collection: Actors;
-    entityClass: typeof Actor;
-    sheetClasses: typeof ActorSheet;
-    typeLabels: any;
+    entityClass: ConstructorOf<Actor>;
+    sheetClasses: Record<string, unknown>;
+    typeLabels: Record<string, string>;
   };
 
   Dice: {
     terms: {
-      c: typeof Coin;
-      d: typeof Die;
-      f: typeof FateDie;
+      c: ConstructorOf<Coin>;
+      d: ConstructorOf<Die>;
+      f: ConstructorOf<FateDie>;
```

Comment:
These should not be changed. They explicitly revert to the one specific type.

Reply:
Why is that?

Reply:
I suspect he may be right. I reverted it for now, since this is mostly about adding the type rather than some sweeping change to CONFIG.